pages/index.tsx

```diff
@@ -1,19 +1,16 @@
 import React, {
   FC,
 } from 'react';
-import dynamic from 'next/dynamic';
 
 // components
 import StaticPage from 'layout/static-page';
 import Head from 'components/head';
-
-const LanguageSelect = dynamic(() => import('../components/language-select'), { ssr: false });
+import Footer from 'components/footer';
 
 const HomePage: FC = () => (
   <StaticPage className="static-custom">
     <Head title="Welcome to Green Energy Data Platform" />
-    <p className="text-5xl bold">Green energy data platform</p>
-    <LanguageSelect />
+    <Footer />
```

Comment:
Shouldn't the footer go inside the `layout` component? As I think, the footer will be on every page, not exclusively on the homepage.

Reply:
sure but we didn't have it at this point